element-connector/src/main/java/org/eclipse/californium/elements/DtlsCorrelationContext.java

```diff
@@ -48,6 +53,14 @@ public DtlsCorrelationContext(String sessionId, String epoch, String cipher) {
 		}
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public Set<String> getMatchingKeys() {
+		return MATCHING_KEYS;
```

Comment:
The point seems for me, that, if new keys are added (e.g. MTU), they may be **not** considered for matching. And "explicit matching keys" may also detect missing entries in only one of the correlation contexts.